src/utils.c

```diff
@@ -55,17 +52,7 @@ ts_pg_timestamp_to_unix_microseconds(PG_FUNCTION_ARGS)
 		ereport(ERROR,
 				(errcode(ERRCODE_DATETIME_VALUE_OUT_OF_RANGE), errmsg("timestamp out of range")));
 
-#ifdef HAVE_INT64_TIMESTAMP
 	microseconds = timestamp + TS_EPOCH_DIFF_MICROSECONDS;
```

Comment:
I don't know that we need this local variable (just directly return this calculation).